chat-commands.js

```diff
@@ -707,7 +713,10 @@ const commands = {
 		}
 		return this.errorReply(`An unknown error occurred while trying to create the room '${title}'.`);
 	},
-	makegroupchathelp: [`/makegroupchat [roomname], [hidden|private] - Creates a group chat named [roomname]. Leave off privacy to default to hidden. Requires global voice or roomdriver+ in a public room to make a groupchat.`],
+	makegroupchathelp: [
+		`/makegroupchat [roomname], [hidden|private] - Creates a group chat named [roomname]. Leave off privacy to default to hidden. Requires global voice or roomdriver+ in a public room to make a groupchat.`,
```

Comment:
Why are we allowing users to choose between "hidden" and "private" in the first place? They should all be hidden.

Reply:
I don't know, it was this way before, want me to change it?

Reply:
Yeah, it was also bad before. :P

Reply:
It was interesting back when we let people choose between `[public|hidden|secret]`. But we got rid of public ones, and honestly in hindsight, we don't want people to make secret ones, so there's no real need for that option right now.

Reply:
``if (target.length > 64) return this.errorReply("Title must be under 32 characters long.");``

I'll change this as well.